translation/service.go

```diff
@@ -14,7 +16,7 @@ import (
 // Service represents the Webpa-Tr1d1um component that translates WDMP data into WRP
 // which is compatible with the XMiDT API.
 type Service interface {
-	SendWRP(*wrp.Message, string) (*common.XmidtResponse, error)
+	SendWRP(*wrp.Message, string,context.Context) (*common.XmidtResponse, error)
```

Comment:
Let's make the context in the different functions be the first argument to follow the Go convention https://golang.org/pkg/context/

Reply:
Made context as the first param of each function